Dockerfile

```diff
@@ -1,27 +1,26 @@
 # Dockerfile # Mapknitter
 # https://github.com/publiclab/mapknitter/
+# This image deploys Mapknitter!
 
 FROM ruby:2.4.4-stretch
-MAINTAINER Sebastian Silva "sebastian@fuentelibre.org"
```

Comment:
Removed since its deprecated - [REF](https://docs.docker.com/engine/deprecated/#maintainer-in-dockerfile)